mock/src/mock/notes.rs

```diff
@@ -192,9 +193,61 @@ pub fn mock_notes(
     )
     .unwrap();
 
+    // note that changes the account vault
+    let note_5_script_ast = ProgramAst::parse(
+        "\
+                    use.miden::sat::note
+                    use.miden::wallets::basic->wallet
+
+                    begin
+                        # read the assets to memory 
+                        push.0 exec.note::get_assets
+                        # => [num_assets, dest_ptr]
+
+                        # assert the number of assets is 3
+                        push.3 assert_eq
+                        # => [dest_ptr]
+
+                        # add the first asset to the vault
+                        padw dup.4 mem_loadw call.wallet::receive_asset dropw
+                        # => [dest_ptr]
+
+                        # add the second asset to the vault
+                        push.1 add padw dup.4 mem_loadw call.wallet::receive_asset dropw
+                        # => [dest_ptr+1]
+
+                        # add the third asset to the vault
+                        push.1 add padw movup.4 mem_loadw call.wallet::receive_asset dropw
+                        # => []
+                    end
+                    ",
+    )
+    .unwrap();
+    let (note_5_script, _) = NoteScript::new(note_5_script_ast, assembler).unwrap();
+    const SERIAL_NUM_8: Word = [Felt::new(29), Felt::new(30), Felt::new(31), Felt::new(32)];
+    let consumed_note_5 = Note::new(
+        note_5_script,
+        &[],
+        &[
+            fungible_asset_1,
+            fungible_asset_3,
+            non_fungible_asset_2(ACCOUNT_ID_NON_FUNGIBLE_FAUCET_ON_CHAIN),
+        ],
+        SERIAL_NUM_8,
+        sender,
+        Felt::ZERO,
```

Comment:
minor nit: here (and in other places in this file), we can use `ZERO` instead of `Felt::ZERO`.